tests/xrt/multi_xclbin/simple.cpp

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+/****************************************************************
+This example illustrates the use of xrt::hw_context for working with
+multiple xclbins
+
+% g++ -g -std=c++17 -I$XILINX_XRT/include -L$XILINX_XRT/lib -o simple.exe simple.cpp -lxrt_coreutil -luuid -pthread
+****************************************************************/
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_kernel.h"
+
+#include "experimental/xrt_hw_context.h"
+
+#include <array>
+#include <atomic>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <list>
+#include <thread>
+#include <vector>
+
+#ifdef _WIN32
+# pragma warning ( disable : 4267 )
+#endif
+
+// Kernel specifics
+// void addone (__global ulong8 *a, __global ulong8 * b, unsigned int  elements)
+// addone(a, b, ELEMENTS)
+// The kernel is compiled with 8 CUs same connectivity.
+static constexpr size_t elements = 16;
```

Comment:
Do you really need all these `static` everywhere in a such single-file example? Same for the functions.
Nowadays the anonymous `namespace` might be more fashionable for this anyway.